configuration/configuration.go

```diff
@@ -245,7 +244,7 @@ func (c *ConfigurationData) checkClusterConfig() error {
 	return nil
 }
 
-func convertAPIURL(apiURL string, newPrefix string, newPath string) (string, error) {
+func ConvertAPIURL(apiURL, newPrefix, newPath string) (string, error) {
```

Comment:
If you want to export this function then move it to `cluster` package (`/cluster/cluster.go`?). It's not a configuration implementation detail anymore.

Reply:
done in 47923fb